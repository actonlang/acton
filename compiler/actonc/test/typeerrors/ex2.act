def f(x,y):
    return x+y

a = f(3,4,5)
