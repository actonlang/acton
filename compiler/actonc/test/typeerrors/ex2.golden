Building file test/typeerrors/ex2.act
  Compiling ex2.act for development
[error]: Incompatible types
     +--> ex2@4:5-4:13
     |
   1 | def f(x,y):
     :     ^ 
     :     `- f is defined here
     :
   4 | a = f(3,4,5)
     :     ^-------
     :     `- too many positional argument(s) in call to f
-----+
